### PR TITLE
Feature modal

### PR DESCRIPTION
**Pull request by OmKakatkar**
* Adds 3 types of modals to the CL
* Please ignore the Toast and List files they are a part of another PR
* Please ignore the broken links they will be added later

[Preview Modal](https://deploy-preview-16--ani-design.netlify.app/docs/modal.html)

### CODE REVIEW

**Comment by MonikaJhingan** [comment]
The code us well indented but please add meta tag for author and description in the head tag of html code which will increase code readability . Rest the usage of rems and percentage is done neatly.

**Comment by shrey27** [comment]
Suggestions:
- In Modal list, list is flowing out of the modal container
- Modals should close by clicking outside them as well
Toast is working fine

Merge this PR post bug fix

**Review by shrey27 — APPROVED** [review]

**Comment by OmKakatkar** [comment]
> Suggestions:
> 
> * In Modal list, list is flowing out of the modal container
> * Modals should close by clicking outside them as well
>   Toast is working fine
> 
> Merge this PR post bug fix

Thanks for the suggestions,
The modal list, is scrollable as you can see below.
This is an intended feature
image

File: css/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.modal {
+	position: fixed;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 25;
```
Comment (tejas-muthya37) [inline css/components/modal.css:9]: Not a fan of arbitrary numbers for z-index. Please analyze your layout well and assign the accurate value.

**Review by tejas-muthya37 — COMMENTED** [review]
Did not find a lot of issues with the code. 3 things that I liked were the consistency in naming, use of CSS variables and use of comments.

**Pull request merged**